src/redux/wallet/actions.js

```diff
@@ -151,21 +153,15 @@ export const withdrawTIME = (amount) => async (dispatch) => {
 //   })
 // }
 
+const paginator = new BlockDataPaginator(new WalletTxsProvider())
+paginator.provider.account = LS.getAccount()
```

Comment:
**Error** use LS.getAccount() directly in your provider code, because it is assumed that it will only return transactions for the current account